style.css

```diff
@@ -0,0 +1,15 @@
+.test{
+    background: red;
+}
+
+.text-list{
+    font-size:14px;
+}
+.video-caption{
+    font-size:17px;
+}
```

Comment:
Nice project . but I will like to give a feedback on your Css.

I think you should add implement the right space and indentation on your stylesheet .
eg. h1 {
background-colour: black;
}